src/modules/mavlink/mavlink_mission.cpp

```diff
@@ -70,14 +76,12 @@ MavlinkMissionManager::MavlinkMissionManager(Mavlink *mavlink) : MavlinkStream(m
 	_action_timeout(MAVLINK_MISSION_PROTOCOL_TIMEOUT_DEFAULT),
 	_retry_timeout(MAVLINK_MISSION_RETRY_TIMEOUT_DEFAULT),
 	_max_count(DM_KEY_WAYPOINTS_OFFBOARD_0_MAX),
-	_dataman_id(0),
-	_count(0),
-	_current_seq(0),
+	_my_dataman_id(0),
 	_transfer_dataman_id(0),
 	_transfer_count(0),
 	_transfer_seq(0),
 	_transfer_current_seq(0),
-	_transfer_partner_sysid(0),
+	_transfer_partner_sysid(255),
```

Comment:
That's a hack - it should default to 0 and should work that way, because its set on transfer init.